CRM/Member/Form/Membership.php

```diff
@@ -1356,6 +1356,10 @@ public function submit() {
     if ($this->_mode) {
       $params['total_amount'] = CRM_Utils_Array::value('total_amount', $formValues, 0);
 
+      //CRM-20264 : Store CC type and number (last 4 digit) during backoffice or online payment
```

Comment:
Issue is at here https://github.com/civicrm/civicrm-core/pull/10234/files#diff-fa0276456bc05f4629f775e6846f62edR1316 where ```$params``` is taking specific attribute if ```record_contribution = TRUE``` which is ONLY available at backoffice form not in live mode. So I found as this fix is the win-win solution.